src/main.rs

```diff
@@ -342,21 +342,22 @@ fn read_tasks() -> Result<Vec<Group>> {
     Ok(tasks)
 }
 
-fn create_process(task: &Task) -> Result<Child> {
+fn create_process(task: &Task, inherit_stdio: bool) -> Result<Child> {
     let current_dir = current_dir()?;
     let working_dir = task.working_dir.as_ref().unwrap_or(&current_dir);
     let mut child = Command::new("sh");
     child.args(["-c", &format!("exec {}", task.cmd)])
         .current_dir(working_dir)
-        .stdin(Stdio::inherit())
-        .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit())
-        .envs(&task.env);
+        .stdin(if inherit_stdio { Stdio::inherit() } else { Stdio::piped() })
+        .stdout(if inherit_stdio { Stdio::inherit() } else { Stdio::piped() })
+        .stderr(if inherit_stdio { Stdio::inherit() } else { Stdio::piped() });
```

Comment:
Please make sure you're using `cargo fmt` before committing. This practice minimizes the amount of formatting changes in new merge requests.